### DISCUSSION TITLE
New Project

### DISCUSSION THREAD

**Discussion by mojidev-py** [mojidev-py/discmoji#38] [Announcements]
I made a better way to tell if something needs to be implemented, inspired from demo's discord api wrapper's project, so i would like feedback and thoughts, and if there's anything you want to add to it, please do

**Comment by ghost** [comment]
you can provide descriptions to your project items to give more context to the pending implementation / fix

( i created the projects to keep track of the pending implementations when i was sick lol, or when i didn't have time to do the idea )

**Reply by mojidev-py** [reply]
got it
